README.md

```diff
@@ -144,13 +144,57 @@ With the newly added `fp8dqrow` scheme, we can bring down the inference latency
 
 </details>
 
+
 ### Trade-offs, trade-offs, and more trade-offs
 
 We know that the table included above is hard to parse. So, we wanted to include a couple of points that are worth noting. 
 
 * Select the quantization technique that gives you the best trade-off between memory and latency. 
 * A quantization technique may exhibit different optimal settings for a given batch size. For example, for a batch size of 4, `int8dq` gives best time without any QKV fusion. But for other batch sizes, that is not the case.
 
+The section below, drives this point home.
+
+#### Higher batch sizes like 16 and above
```

Comment:
does integer quant gives speedup or is it mostly slow down right now? like int4wo, int8wo, int8dyn?

Reply:
It does but FP is just faster at the moment.